mineral-ui/mineral-ui
# chore(dependencies): Update prettier and eslint packages

### Description

Update prettier and eslint packages
* Ran `npm run format` to apply prettier updates

### Motivation and context

Updating dependencies...

### How to test

1. Check out branch.
2. `npm test` results in no linter errors.  Note that I had to restart Atom in order for it to recognize updates.

### Types of changes

- Other (provide details below)

Updated dependencies. Stylistic formatting... 

### Checklist

* [x] Rendering performance (initial load time & 60fps) and [perceived performance] - [n/a](http://blog.teamtreehouse.com/perceived-performance) considered
